projects/seamless/index.js

```diff
@@ -1,10 +1,53 @@
-const { aaveExports } = require("../helper/aave");
-const methodologies = require("../helper/methodologies");
+const { getLogs2 } = require("../helper/cache/getLogs");
```

Comment:
please use this file for seamless v1, and add a file with a different path for v2